ext/mysqli/mysqli.stub.php

```diff
@@ -470,35 +444,25 @@ public function getIterator(): Iterator;
 
 class mysqli_stmt
 {
-    /** @var int|string|null */
-    public $affected_rows;
+    public int|string|null $affected_rows;
 
-    /** @var int|string|null */
-    public $insert_id;
+    public int|string $insert_id;
 
-    /** @var int|string|null */
-    public $num_rows;
+    public int|string $num_rows;
 
-    /** @var int|string|null */
-    public $param_count;
+    public int $param_count;
 
-    /** @var int|string|null */
-    public $field_count;
+    public int $field_count;
 
-    /** @var int|string|null */
-    public $errno;
+    public int $errno;
 
-    /** @var string|null */
-    public $error;
+    public string $error;
 
-    /** @var array|null */
-    public $error_list;
+    public array $error_list;
 
-    /** @var string|null */
-    public $sqlstate;
+    public string $sqlstate;
 
-    /** @var int|null */
-    public $id;
+    public ?int $id;
```

Comment:
This is an undocumented property. We should probably consider dropping it at some point of time. For the moment we can make only int with assert